trino/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Can you populate this file with the metrics this integrations submits? See https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion2#metrics-metadata-file for more information